toolchain/check/convert.cpp

```diff
@@ -1207,18 +1207,18 @@ auto ConvertCallArgs(Context& context, SemIR::LocId call_loc_id,
   for (auto implicit_param_id : implicit_param_refs) {
     auto addr_pattern =
         context.insts().TryGetAs<SemIR::AddrPattern>(implicit_param_id);
-    auto [param_id, param] = SemIR::Function::GetParamFromParamRefId(
-        context.sem_ir(), implicit_param_id);
-    if (param.name_id == SemIR::NameId::SelfValue) {
+    auto param = SemIR::Function::GetParamFromParamRefId(context.sem_ir(),
```

Comment:
Here and elsewhere, it seems like this should be named `param_info` and/or its type should be explicit; otherwise it's too easy to mistakenly assume the type is `Param`.

Reply:
Done.